platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/address/AddressBookUtils.java

```diff
@@ -44,9 +44,14 @@
  *     <li>memo field (optional)</li>
  * </ul>
  * Example: `address, 22, node22, node22, 1, 10.10.11.12, 5060, 212.25.36.123, 5060, memo for node 22`
+ * <p>
+ * The last line of the config.txt address book contains the nextNodeId value in the form of: `nextnodeid, 23`
  */
 public class AddressBookUtils {
 
+    public static final String ADDRESS_KEYWORD = "address";
+    public static final String NEXT_NODE_ID_KEYWORD = "nextnodeid";
```

Comment:
Should be `nextNodeId`

Reply:
this one can't be `nextNodeId` because the parsing algorithm makes everything lowercase before the switch on the keyword at the start of each line.

Reply:
I'm going to try to make it work with the camel case.

Reply:
So `addressBookConfigText` will serialize the string with different case than the manually generated file?